### PR TITLE
update documentation

### LINKED ISSUE #337: [Question] Where can I set the jacknife or CR0 (sandwich) robust covariance?
Dear Authors,

I'm a bit lost.. how to set the sandwich covariance estimator (can be just the classic CR0)? I urgently need it during the analysis and just realized I don't know how to set it.

This links says it's done: https://github.com/openpharma/mmrm/issues/241

Is this via " vcov = "Empirical-Bias-Reduced" ?
(taken from this: https://github.com/openpharma/mmrm/blob/d92e8b1073aa37010ea33d48d9cfc4f0c5ec4228/tests/testthat/test-empirical.R#L342 )

or should I take it from here and set  "vcov=Empirical"? vcov="Jacknife"?
https://openpharma.github.io/mmrm/main/reference/mmrm_control.html

And if I wanted the "Jackknife", then how should I set it?

Moreover, I put Kenward-Roger-Linear into search text box 
and got this:
https://openpharma.github.io/mmrm/articles/introduction.html?q=Kenward-Roger-Linear%20#variance-covariance-for-coefficients

also when typed "Jackinife" to find the details:
https://openpharma.github.io/mmrm/articles/coef_vcov.html?q=Jackknife%20#jackknife-covariance
obraz

PS: if the Kenward-Roger-Linear is known to be more SAS-compliant, will you make it the default setting in the nearest release? It's easy to miss it. It would be great if the maximum compliance was done with reasonably minimal effort - by setting some major options default.

### PR DESCRIPTION
**Pull request by clarkliming**
close #337

### CODE REVIEW

File: vignettes/coef_vcov.Rmd [hunk 1]
```diff
@@ -52,7 +52,7 @@ Where $W$ is the block diagonal matrix of inverse of covariance matrix of $\epsi
 
 ## Empirical Covariance
```
Comment (danielinteractive) [inline vignettes/coef_vcov.Rmd:53]: Can we also in this vignette include the syntax for how to request this in `mmrm`?

**Review by danielinteractive — COMMENTED** [review]
Thanks @clarkliming , much appreciated -
on top, can we please also in https://github.com/openpharma/mmrm/blob/main/vignettes/subsections/intro-customizations.Rmd#L168 include a code example and also mention how the methods are called in code?

**Review by danielinteractive — COMMENTED** [review]
Thanks @clarkliming , can you also add an overview of how the `vcov` methods are called? Maybe also the same for `method`. So that somebody who reads this intro vignette knows what options are available

**Comment by danielinteractive** [comment]
@clarkliming let me know if you don't have time then I can also finish this

**Comment by clarkliming** [comment]
I think there are already included 

(methods customization)
https://github.com/openpharma/mmrm/blob/972d43413c5067c5f6d7c7fc46fcd69ed52fd7a6/vignettes/subsections/intro-customizations.Rmd#L124

and

(vcov customization)
https://github.com/openpharma/mmrm/blob/972d43413c5067c5f6d7c7fc46fcd69ed52fd7a6/vignettes/subsections/intro-customizations.Rmd#L185

**Review by danielinteractive — APPROVED** [review]
Thanks @clarkliming !

**Pull request merged**